sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -275,6 +275,10 @@ def set_resolved_hostname(self):
 
     @property
     def resolved_hostname(self):
+        # ensure _resolved_hostname is never None when referenced by other parts
+        # of this check. If it falls back to None, this will result in gaps in data
+        if self._resolved_hostname is None:
+            return self.set_resolved_hostname()
```

Comment:
set_resolved_hostname has no return
```suggestion
            self.set_resolved_hostname()
```

Reply:
oh yeah that was a typo, i fixed it in my last commit